src/main/java/com/google/webauthn/gaedemo/servlets/FinishGetAssertion.java

```diff
@@ -120,6 +143,12 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
     // }
 
     Server.verifyAssertion(cred, currentUser, session, savedCredential);
+    if (uvm != null) {
+      savedCredential.setUserVerificationMethod(uvm);
+    } else {
+      savedCredential.setUserVerificationMethod("unknown");
```

Comment:
Could we just ignore this case and not set anything if there is no uvm returned? If someone doesn't request/receive UVM, I don't think we should show anything.

Reply:
Sounds reasonable.